en/04_Changelogs/5.2.0.md

```diff
@@ -157,6 +157,12 @@ Both of these fields include a `setIsLazyLoaded()` method which will load a limi
 
 Note that these are both powered by react components, and are only intended to be used within the CMS. If you want to use them on the front-end of your project you will need to provide your own templates and JavaScript implementation for them.
 
+#### Auto scaffolding of `has_one` relations
+
+A `SearchableDropdownField` will now be used when automatically scaffolding `has_one` relations on a form in the CMS. Previously a `DropdownField` was used, and when there were over 100 items a [`NumericField`](api:SilverStripe\Forms\NumericField) was used which was not user friendly.
+
+Previously the [`DBForeignKey.dropdown_field_threshold`](api::SilverStripe\ORM\FieldType\DBForeignKey->dropdown_field_threshold) config property was used as the threshold of the number of options to decide when to switch between auto-scaffolding a `DropdownField` and a `NumericField`. This configuration property is now used as the threshold of the number of options to decide when to start using lazy-loading for the `SearchableDropdownField`.
```

Comment:
```suggestion
Previously the [`DBForeignKey.dropdown_field_threshold`](api:SilverStripe\ORM\FieldType\DBForeignKey->dropdown_field_threshold) config property was used as the threshold of the number of options to decide when to switch between auto-scaffolding a `DropdownField` and a `NumericField`. This configuration property is now used as the threshold of the number of options to decide when to start using lazy-loading for the `SearchableDropdownField`.
```
This was in https://github.com/silverstripe/developer-docs/pull/425#discussion_r1430677061 as well but got lost in the property syntax stuff.

Reply:
Updated